websites/A/AnimeVost/metadata.json

```diff
@@ -0,0 +1,67 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.7",
+	"author": {
+		"id": "296716315547402240",
+		"name": "ZebroUnion"
+	},
+	"service": "AnimeVost",
+	"altnames": [
+		"АнимеВост"
+	],
+	"description": {
+		"en": "AnimeVost - Best website to watch anime online for free in Russian.",
+		"ru": "AnimeVost - Лучший сайт для просмотра аниме онлайн бесплатно на Русском."
+	},
+	"url": [
+		"animevost.org",
+		"www.animevost.org",
+		"v2.vost.pw"
+	],
+	"version": "1.0.0",
+	"logo": "https://i.imgur.com/wStmaRE.png",
+	"thumbnail": "https://i.imgur.com/MeWpyAW.png",
+	"color": "#ffb84d",
+	"category": "anime",
+	"tags": [
+		"animevost",
+		"vost",
+		"anime",
+		"аниме",
+		"анимевост",
+		"вост",
+		"animegost"
+	],
+	"iframe": true,
```

Comment:
Please also add an `iFrameRegExp` field, unless the iframes used on the website use domains already contained in the `url` field.

Reply:
iFrame uses domains already contained in the `url` field.
Here is some screenshots (same links on `animevost.org`):
![image](https://user-images.githubusercontent.com/57961585/233858568-2ddd896d-5211-47b2-bf57-d14838501244.png)
![image](https://user-images.githubusercontent.com/57961585/233858630-e531d4e7-010a-4dcd-826d-193c24ebadc6.png)